publish/publish-type.proj

```diff
@@ -48,16 +48,18 @@
     <!-- TODO: We should also think about pulling down the checksums or recomputing them -->
     <PropertyGroup>
       <DownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(BinDir)PackageDownload/</DownloadDirectory>
+      <!-- Must add a / suffix or else a stable build will end up downloading too much (e.g. everything prefixed with 2.1.0-preview2) -->
+      <DownloadBlobNamePrefix>$(BinariesRelativePath)/</DownloadBlobNamePrefix>
```

Comment:
Do we need to add a check before adding it? What happens if it already has one in some cases?

Reply:
Perhaps we should actually add it to BinariesRelativePath property itself.

Reply:
Yeah I think it would be best to just add the slash at https://github.com/dotnet/core-setup/blob/2ab41fcbe3e1dde2f7deebdb15aabc258cff1130/dir.props#L120